src/web/app/page.component.html

```diff
@@ -107,7 +109,6 @@ <h5 style="text-align: center;"><i class="fas fa-exclamation-circle"></i>&nbsp;O
       </div>
     </div>
   </div>
-  <h1 *ngIf="pageTitle && isValidUser">{{ pageTitle }}</h1>
```

Comment:
Why is this moved up and why the `isValidUser` condition check is removed?

Reply:
This was originally moved up to show check the pageTitle in `containsExpectedPageContents()` despite showing error message in E2E test. Since that test is not required, I will remove this as well